README.md

```diff
@@ -1,5 +1,8 @@
 # APsystems EZ1 -  Python Library
 
+[![apsystems-ez1 @ PyPi](https://img.shields.io/pypi/v/energymeter2mqtt?label=apsystems-ez1%20%40%20PyPi)](https://pypi.org/project/apsystems-ez1/)
+[![License GPL-3.0-or-later](https://img.shields.io/pypi/l/energymeter2mqtt)](https://github.com/SonnenladenGmbH/APsystems-EZ1-API/blob/main/LICENSE)
+
```

Comment:
Why does it mention energymeter2mqtt in the shields source link?

Reply:
Oh, sorry. Copy&paste error. I missed to replace the string.

Reply:
fixed